Sources/AnimalKingdomAPI/AnimalKingdomAPI/TestMocks/Bird+Mock.graphql.swift

```diff
@@ -47,7 +47,7 @@ public extension Mock where O == Bird {
     _setScalar(laysEggs, for: \.laysEggs)
     _setEntity(owner, for: \.owner)
     _setList(predators, for: \.predators)
-    _setEntity(skinCovering, for: \.skinCovering)
+    _setScalar(skinCovering, for: \.skinCovering)
```

Comment:
What's causing these to change? `GraphQLEnum` is simply a custom scalar but how did we end up with `_setEntity` to begin with?

Reply:
I originally had it mistakenly grouped with `entity` instead of `scalar` in the switch statement when determining the function descriptor, I believe I fixed that before the initial PR creation, but hadn't regenerated the code for AnimalKingdomAPI